definitions/npm/reselect_v3.x.x/flow_v0.34.x-/reselect_v3.x.x.js

```diff
@@ -1,85 +1,144 @@
-declare module 'reselect' {
-  declare type Selector<TState, TProps, TResult> = {
-      (state: TState, props: TProps, ...rest: any[]): TResult;
+declare module "reselect" {
+  declare type Selector<-TState, TProps, TResult> = {
```

Comment:
This is the only meaningful change. This comes from @calebmer in https://twitter.com/calebmer/status/915669046191824896

All other changes are prettier's work :)